easybuild/easyconfigs/a/adios/adios-2.6.0-CrayGnu-20.08.eb

```diff
@@ -0,0 +1,52 @@
+# contributed by Jean Favre (CSCS)
+
+easyblock = 'CMakeMake'
+
+name = "adios"
+version = '2.6.0'
+
+homepage = 'https://csmd.ornl.gov/software/adios2'
+description = """The Adaptable IO System (ADIOS) provides a simple, flexible way for scientists 
+to describe the data in their code that may need to be written, read, or processed outside of the running simulation"""
+
+toolchain = {'name': 'CrayGNU', 'version': '20.08'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['https://github.com/ornladios/ADIOS2/archive/']
+sources = ["release_26.zip"]
+
+builddependencies = [
+    ('CMake', '3.14.5', '', True),
+]
+
+configopts  = '-DCMAKE_BUILD_TYPE=Release '
+configopts += "-DCMAKE_Fortran_FLAGS='-O2 -DNDEBUG -fPIC -fno-math-errno ' "
+configopts += "-DCMAKE_C_FLAGS='-DNDEBUG -fPIC -std=c11' "
+configopts += "-DCMAKE_CXX_FLAGS='-O2 -DNDEBUG -fPIC -std=gnu++11 ' "
+#configopts  = '-DCMAKE_BUILD_TYPE=RelWithDebInfo '
+configopts  += '-DBUILD_SHARED_LIBS:BOOL=ON -DADIOS2_BUILD_EXAMPLES:BOOL=OFF -DADIOS2_BUILD_TESTING:BOOL=OFF '
```

Comment:
I think `ADIOS2_USE_SST=ON` should be added to enable proper working of in-situ features (c.f. https://adios2.readthedocs.io/en/latest/setting_up/setting_up.html?highlight=blosc#cmake-options)

Reply:
Done in my PR.